website/server/libs/cron.js

```diff
@@ -130,6 +130,14 @@ function trackCronAnalytics (analytics, user, _progress, options) {
 
 function awardLoginIncentives (user) {
   if (user.loginIncentives > 50) return;
+
+  //  Remove old noitification if it exists
+  user.notifications
+    .toObject()
+    .find((notif, index) => {
```

Comment:
To make use of find to short circuit the loop, you should return a truthy value when you've done the required operation. The way you have it now, it will continue to search through the array after it has found the notification.